src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -61,8 +61,10 @@ const BrowserTrackConfig: FunctionComponent<BrowserTrackConfigProps> = (
 
   const selectedCog = props.selectedCog || '';
 
-  const shouldShowTrackName = trackConfigNames[selectedCog] || false;
-  const shouldShowTrackLabels = trackConfigLabel[selectedCog] || false;
+  const shouldShowTrackName =
+    trackConfigNames[selectedCog] !== false ? true : false;
+  const shouldShowTrackLabels =
+    trackConfigLabel[selectedCog] !== false ? true : false;
```

Comment:
How is this different from `trackConfigLabel[selectedCog] || false`? Is `trackConfigLabel[selectedCog]` a boolean?

Reply:
We need to consider `undefined` as true.  `trackConfigLabel[selectedCog]` will be empty by default.

Reply:
No we can't do that as `undefined || true`  and `false || true` will always return true.